docs/faq.rst

```diff
@@ -0,0 +1,218 @@
+*******************************************************************************
+Frequently asked questions
+*******************************************************************************
+
+HIP Support
+===========
+
+What CUDA features does HIP support?
+----------------------------------------
+
+The :doc:`hipify:tables/CUDA_Runtime_API_functions_supported_by_HIP`
+and :doc:`hipify:tables/CUDA_Driver_API_functions_supported_by_HIP`
+pages detail what CUDA APIs are supported and what the equivalents are.
+The `HIP API documentation <doxygen/html/index.html>`_ describes each API and
+its limitations, if any, compared with the equivalent CUDA API.
+
+The kernel language features are documented in the :doc:`/reference/cpp_language_extensions` page.
+
+What libraries does HIP provide?
+--------------------------------
+
+HIP provides key math and AI libraries. The full list can be found here: :doc:`rocm:reference/api-libraries`
+
+What hardware does HIP support?
+-------------------------------
+
+* For AMD platforms, see the :doc:`GPU support on Linux <rocm-install-on-linux:reference/system-requirements>`
+  or :doc:`GPU support on Windows <rocm-install-on-windows:reference/system-requirements>`.
+* For NVIDIA platforms, HIP requires unified memory and should run on any
+  device with compute capability 5.0 or higher.
+
+What operating systems does HIP support?
+----------------------------------------
+
+The supported operating systems are listed in the :doc:`rocm:compatibility/compatibility-matrix`
+
+CUDA and OpenCL
+===============
+
+Is HIP a drop-in replacement for CUDA?
+--------------------------------------
+
+The `HIPIFY <https://github.com/ROCm/HIPIFY>`_ tools can automatically convert
+almost all runtime code. Most device code needs no additional conversion since
+HIP and CUDA have similar names for math and built-in functions. HIP code
+provides the same performance as native CUDA code, plus the benefits of running
+on AMD platforms.
+
+Additional porting may only be required to deal with architecture feature
+queries or with CUDA capabilities that HIP doesn't support.
+
+How does HIP compare with OpenCL?
+---------------------------------
+
+HIP offers several benefits over OpenCL:
+
+* Device code can be written in modern C++, including templates, lambdas, classes and so on.
+* Host and device code can be mixed in the source files.
+* The HIP API is less verbose than OpenCL and is familiar to CUDA developers.
+* Porting from CUDA to HIP is significantly easier than porting from CUDA to OpenCL.
+* HIP uses the best available development tools on each platform: ``amdclang++`` for AMD GPUs or ``nvcc``
+  for NVIDIA GPUs, and profilers like ``omniperf`` or ``Nsight Systems``.
+* HIP provides pointers and host-side pointer arithmetic.
+* HIP provides device-level control over memory allocation and placement.
+* HIP offers an offline compilation model.
+
+How does porting CUDA to HIP compare to porting CUDA to OpenCL?
```

Comment:
Link this page?
https://rocm.docs.amd.com/projects/HIP/en/latest/reference/terms.html